terraform/environments/data-platform/lambda.tf

```diff
@@ -402,4 +402,32 @@ module "preview_data_lambda" {
   }
 }
 
+module "delete_table_for_data_product_lambda" {
+  source                         = "github.com/ministryofjustice/modernisation-platform-terraform-lambda-function?ref=a4392c1" # ref for V2.1
+  application_name               = "delete_schema"
```

Comment:
how come it's `delete_schema` here and `delete_table` everywhere else?

Reply:
updated